cli/src/pcluster/validators/fsx_validators.py

```diff
@@ -222,3 +222,16 @@ def _validate(self, auto_import_policy, import_path):
             bucket = get_bucket_name_from_s3_url(import_path)
             if AWSApi.instance().s3.get_bucket_region(bucket) != get_region():
                 self._add_failure("FSx auto import is not supported for cross-region buckets.", FailureLevel.ERROR)
+
+
+class FsxDescribeVolumesValidator(Validator):
+    """Backup id validator."""
```

Comment:
Why Backup?

Reply:
Sorry, this is a remnant from a reference. So, I'll change this.